.docker/install-plugin-theme.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Install plugins
+echo "Installing plugins"
+wp --allow-root plugin install akismet buddypress bbpress jetpack wp-multibyte-patch wordpress-importer polldaddy liveblog wp-super-cache custom-content-width camptix-network-tools email-post-changes tagregator supportflow camptix-pagseguro camptix-payfast-gateway camptix-trustpay camptix-trustcard camptix-mercadopago camptix-kdcpay-gateway campt-indian-payment-gateway camptix
+
+# Install themes
+echo "Installing themes"
+wp --allow-root theme install twentyten twentyeleven twentytwelve twentythirteen twentyfourteen twentyfifteen twentysixteen twentyseventeen twentyeighteen p2
```

Comment:
```suggestion
wp --allow-root theme install twentyten twentyeleven twentytwelve twentythirteen twentyfourteen twentyfifteen twentysixteen twentyseventeen twentynineteen p2
```

`MUST` - There wasn't a `twentyeighteen` theme, we skipped that year, but there is a `twentynineteen`

```sh
Installing Twenty Seventeen (2.2)
Downloading installation package from https://downloads.wordpress.org/theme/twentyseventeen.2.2.zip...
The authenticity of twentyseventeen.2.2.zip could not be verified as no signature was found.
Unpacking the package...
Installing the theme...
Theme installed successfully.
Warning: twentyeighteen: Theme not found
Warning: The 'twentyeighteen' theme could not be found.
***************** ^
Installing P2 (1.5.8)
Downloading installation package from https://downloads.wordpress.org/theme/p2.1.5.8.zip...
The authenticity of p2.1.5.8.zip could not be verified as no signature was found.
Unpacking the package...
Installing the theme...
Theme installed successfully.
Error: Only installed 9 of 10 themes.
***************** ^
```

Reply:
Done in https://github.com/WordPress/wordcamp.org/pull/113/commits/41867e7168934f7be3093a8ade78a9c1d3604520